.travis.yml

```diff
@@ -21,7 +21,6 @@ jobs:
     - "./scripts/pullContracts.sh"
     - "./scripts/migrate.sh rinkeby"
     - "./scripts/migrate.sh kovan"
-    - "./scripts/migrate.sh ropsten"
```

Comment:
why are we removing ropsten?

Reply:
because it fails, no funds and we will revisit the entire deploy precedure soon. I will spec out the development release cycle today.
@vimukthi-git do you need published addresses for this?

Reply:
nope, was just curious about the update.